src/poetry_plugin_export/command.py

```diff
@@ -82,9 +82,22 @@ def handle(self) -> None:
                 "</warning>"
             )
 
+        # Checking extras
+        extras = []
+        for extra in self.option("extras"):
+            if " " in extra:
```

Comment:
I realise I pointed you at code like this to copy, but it occurs to me now that the two branches are unnecessary, this first branch works fine for both cases.

Reply:
in fact come to think of it isn't this whole thing just
```python
extras = extra.split()
```

Reply:
I changed it to a double comprehension, 🤞 I don't get sent straight to hell